docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -1391,6 +1391,27 @@ UDP
   TCP/UDP
     Not implemented ("-").
 
+%PATH(X:Y):Z%
```

Comment:
What is Z here? It's not documented

Reply:
Z is max length which is common paramter for lots of string formatters.